pkgs/development/libraries/libusb1/default.nix

```diff
@@ -24,9 +26,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-OtzYxWwiba0jRK9X+4deWWDDTeZWlysEt0qMyGUarDo=";
   };
 
-  outputs = [ "out" "dev" ];
+  outputs = [ "out" "dev" ] ++ lib.optionals withDocs [ "doc" ];
 
-  nativeBuildInputs = [ pkg-config autoreconfHook ];
+  nativeBuildInputs = [
+    pkg-config
+    autoreconfHook
+  ] ++ lib.optional withDocs doxygen;
```

Comment:
```suggestion
  ] ++ lib.optionals withDocs [ doxygen ];
```

Reply:
Whoops, sorry for missing this one; fixed!